src/IO.cpp

```diff
@@ -196,8 +196,9 @@ void WriteOutputsToFiles(int smc, int herd_id, int Nparticle, int nPar,
 		output_step	<< particleList[kk].weight<< '\n';
 		
 		for (unsigned int var = 0; var < particleList[kk].simu_outs.size(); ++var) {
-//				cout  << particleList[kk].iter << ", " << var << ", " <<  particleList[kk].simu_outs[var] << ", " <<  particleList[kk].death_outs[var] << '\n';
-			output_simu << particleList[kk].iter << ", " << var << ", " <<  particleList[kk].simu_outs[var] << ", " <<  particleList[kk].death_outs[var] << '\n';
+//				cout  << particleList[kk].iter << ", " << var << ", " <<  particleList[kk].simu_outs[var] << ", " <<  particleList[kk].hospital_death_outs[var] << '\n';
+			output_simu << particleList[kk].iter << ", " << var << ", " <<  particleList[kk].simu_outs[var] << ", ";
```

Comment:
I would have the entire print in a single statement, and break it across lines. Having two statements makes it superficially look as though it's outputting to two lines, which is a bit confusing i.e.

```
output_simu << particleList[kk].iter << ", " << var << ", " <<  particleList[kk].simu_outs[var] << ", " << 
    particleList[kk].hospital_death_outs[var] << ", " << particleList[kk].death_outs[var] << '\n';
```